Dockerfile.template

```diff
@@ -23,15 +23,15 @@ RUN echo 'APT::Get::Assume-Yes "true";' > /etc/apt/apt.conf.d/90circleci && \
 	&& \
 	locale-gen en_US.UTF-8 && \
 	rm -rf /var/lib/apt/lists/* && \
-
+	\
```

Comment:
This change isn't core to this PR, but the docker build was issuing a warning about unsafe blank lines and the fact that the warning would turn into error in later docker releases.